app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAboutRobot.kt

```diff
@@ -160,6 +172,30 @@ private fun assertSupport() {
     Espresso.pressBack()
 }
 
+private fun assertCrashes() {
+
+    browserScreen {
+    }.openThreeDotMenu {
+    }.openSettings {
+    }.openAboutFirefoxPreview {
+    }
+
+    if (!onView(withText("Crashes")).isVisibleForUser()) {
+        onView(withId(R.id.about_layout)).perform(ViewActions.swipeUp())
+    }
+
+    onView(withText("Crashes"))
+        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+        .perform(click())
+
+    onView(withSubstring("No crash reports have been submitted"))
+        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+
+    for (i in 1..3) {
+        Espresso.pressBack()
+    }
```

Comment:
I would assert that we're back on the first view after.

Reply:
Hi @AaronMT , made the changes, successfully ran 20x on Firebase